orchestrator-salt/src/main/resources/salt/salt/disks/service/scripts/mount-instance-storage.j2

```diff
@@ -52,26 +52,9 @@ mount_all_sequential() {
     local return_value=0
     log $LOG_FILE "mounting storage devices ${device_uuids_arr[@]}"
     local hadoop_fs_dir_counter=1
-    if [[ $MOUNT_PATH = *"ephfs"* ]]; then
```

Comment:
Why is it safe to delete this code, and references to $MOUNT_PATH ?
Are non ephemeral drives being mounted elsewhere ?

Will this change also mean that adding an identical node to a hostgroup can end up with the new node having differently named mount points as compared to other nodes in the system ? (Other nodes in this scenario would have been deployed prior to this change)

Reply:
It is safe to delete the other paths because EBS volumes are mounted using a file called mount-disks.j2. This specific file is used to mount only ephemeral disks.

As for creating a new node after the code is deployed, yes it will create a mismatch in the mounted paths.

Reply:
But i thought this is the desired outcome as we need all the ephemeral disks to be mounted at /ephfs* path. But it will not be a problem where the node has both EBS and ephemeral disks - only the nodes which contain just ephemeral disks because the old code will have mounted them at /fs*

Reply:
> As for creating a new node after the code is deployed, yes it will create a mismatch in the mounted paths.
If it creates a mismatch in the mounted paths within a cluster - then, no, this is not a safe change to make. This is the bit in the writeup about being extremely careful about making a change of this kind.

Please see my overarching comment outside of the CR - https://github.com/hortonworks/cloudbreak/pull/14375#issuecomment-1461757324